node_cli/cli/sync_node.py

```diff
@@ -36,11 +38,31 @@ def sync_node():
     pass
 
 
-@sync_node.command('init', help="Initialize sync SKALE node")
+@sync_node.command('init', help=TEXTS['init']['help'])
 @click.argument('env_file')
+@click.option(
```

Comment:
Not sure that I understood the reason, why new options are not a part of config. What the difference between --archive and ENDPOINT in the context of node setup?

Reply:
these options can't be changed after the init - it would be confusing to keep them in the .env